### DISCUSSION TITLE
Warning: Provided 'plot_folder' doesn't exist

### DISCUSSION THREAD

**Discussion by alamnaushad290** [facebookexperimental/Robyn#702] [Q&A]
I'm trying to use the robyn_outputs() function in R to export some plots and files to a specific folder using the plot_folder argument. However, I keep getting a warning message that says "Provided 'plot_folder' doesn't exist". I've checked that the folder exists and that the name is spelled correctly, but the warning persists. What could be causing this issue?

**Comment by laresbernardo** [comment]
Hi! Probably you're not in the correct working directory. We run dir.exists() to check that the directory exists. Note there are different ways of setting them: "~/Desktop", "/Users/bernardolares/Desktop", ... Probably you're in a working directory and trying to find a directory that's not within that directory. Check your `getwd()`.

**Reply by sab26699** [reply]
Hi, I am still having trouble with this. As advised, I ran getwd() and dir.exists() to get an updated directory. Next, I verified that it exists. My result is below: 

```
> getwd()
[1] "/Users/first.last"

> dir.exists("/Users/first.last")
[1] TRUE
```

Here's the code I'm using when I try to export plots:
```
OutputCollect <- robyn_outputs(
  InputCollect, OutputModels,
  pareto_fronts = "auto", # automatically pick how many pareto-fronts to fill min_candidates (100)
  # min_candidates = 100, # top pareto models for clustering. Default to 100
  # calibration_constraint = 0.1, # range c(0.01, 0.1) & default at 0.1
  csv_out = "pareto", # "pareto", "all", or NULL (for none)
  clusters = TRUE, # Set to TRUE to cluster similar models by ROAS. See ?robyn_clusters
  export = create_files, # this will create files locally
  plot_folder = "/Users/first.last" # path for plots exports and files creation
  plot_pareto = create_files # Set to FALSE to deactivate plotting and saving model one-pagers
)
print(OutputCollect)
```

After using the directory from getwd(), I'm now getting a new error:

```
Error: unexpected symbol in:
"  plot_folder = "/Users/first.last" # path for plots exports and files creation
  plot_pareto"
```

I've tried also using "~/first.last" and get the same error. Any idea what I'm doing wrong? It's probably very simple but I've looked at it for too long and can't see an issue.

**Reply by laresbernardo** [reply]
Have you tried a path without a dot in it?
